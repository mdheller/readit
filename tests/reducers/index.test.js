import reducer from '../../src/reducers';

describe('reducer', () => {
  it('matches the snapshot', () => {
    expect(reducer).toMatchSnapshot();
  });
});
