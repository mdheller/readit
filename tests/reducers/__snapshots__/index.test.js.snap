// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducer matches the snapshot 1`] = `[Function]`;
